.gitignore.txt

```diff
@@ -0,0 +1,2 @@
+indexhtml.txt
+styles.css
```

Comment:
why do you need these files? just delete them. you're using git. if you'll need to take a look at them again you can always go back to the old commits